tests/e2e/backup_restore_suite_test.go

```diff
@@ -240,10 +240,10 @@ var _ = Describe("AWS backup restore tests", func() {
 			Expect(err).ToNot(HaveOccurred())
 			Expect(succeeded).To(Equal(true))
 
-			if brCase.BackupRestoreType == RESTIC && nsRequiresResticDCWorkaround {
-				// run the restic post restore script if restore type is RESTIC
-				log.Printf("Running restic post restore script for case %s", brCase.Name)
-				err = RunResticPostRestoreScript(restoreName)
+			if brCase.BackupRestoreType == RESTIC || brCase.BackupRestoreType == KOPIA && nsRequiresResticDCWorkaround {
+				// run the restic /kopia post restore script if restore type is RESTIC or KOPIA
```

Comment:
This needs to be run any time you have DCs with volumes attached, regardless of B/R type.

Reply:
@sseago could you please be more precise here? My understanding from your comment is that I should remove check for the BackupRestoreType and extend nsRequiresResticDCWorkaround to include ones that has volumes attached only? What about the backwards compatibility, or it doesn't matter as it won't be backported to 1.2 ?

Reply:
This is a change in behavior for 1.3 vs. 1.2. For e2e tests, we're only ever working with backups from the same velero version, so backwards compatibility isn't a concern within the test. As for the plugin changes, the functionality change involves setting additional annotations on backup, so the restore plugin uses the presence of those annotations to trigger the changed behavior, so it should all be backwards-compatible with restoring 1.2 backups.